CHANGELOG.md

```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
+*    Query deprecated since version 4.20. Use Query instead.
+*    QueryTask deprecated since version 4.20. Use query instead.
+*    RasterData deprecated since version 4.20. Use RasterData instead.
+*    RelationParameters deprecated since version 4.20. Use RelationParameters instead.
+*    RelationshipQuery deprecated since version 4.20. Use RelationshipQuery instead.
+*    RouteParameters deprecated since version 4.20. Use RouteParameters instead.
+*    RouteResult deprecated since version 4.20. Use RouteResult instead.
+*    RouteTask deprecated since version 4.20. Use route instead.
+*    SceneView.constraints.collision deprecated since version 4.8. Use Ground.navigationConstraint instead.
+*    ServiceAreaParameters deprecated since version 4.20. Use ServiceAreaParameters instead.
+*    ServiceAreaSolveResult deprecated since version 4.20. Use ServiceAreaSolveResult instead.
+*    ServiceAreaTask deprecated since version 4.20. Use serviceArea instead.
+*    SizeVariable.expression deprecated since version 4.2. Use SizeVariable.valueExpression instead.
+*    Slider.labelsVisible deprecated since version 4.15. Use Slider.visibleElements.labels instead.
+*    Slider.rangeLabelsVisible deprecated since version 4.15. Use Slider.visibleElements.rangeLabels instead.
+*    StatisticDefinition deprecated since version 4.20. Use StatisticDefinition instead.
+*    symbolPreview.renderPreviewHTML deprecated since version 4.11. Use symbolUtils.renderPreviewHTML instead.
+*    symbolPreview deprecated since version 4.11. Use symbolUtils instead.
+*    Task deprecated since version 4.20.
+*    TimeSlider.values deprecated since version 4.20. Use timeExtent instead.
+*    TimeSliderViewModel.values deprecated since version 4.20. Use timeExtent instead.
+*    TrimExtendParameters deprecated since version 4.20. Use TrimExtendParameters instead.
+*    widget.renderable deprecated since version 4.19. All properties are automatically tracked now and don't need to be decorated with this decorator.
+*    `SmartMapping.params.basemap` deprecated since version 4.13. Use `view` instead.
+*    The light-blue, dark-blue, light-green, dark-green, light-purple, dark-purple, light-red, dark-red are deprecated since 4.19. Please use `light` or `dark` instead, or create your own theme.
+
 
 BELOW ARE THE MANUALLY ADDED DEPRECATED CLASSES, PROPERTIES, METHODS, EVENTS
 
-*	decorators.cast(classFunction) deprecated since version 4.14. Parameter decorators won't be supported by JavaScript decorators.
-*	LabelClass.labelExpressionInfo.value deprecated since version 4.5. Use expression instead.
-*	SceneView.constraints.collision deprecated since version 4.8. Use Ground.navigationConstraint instead.
-*	SmartMapping.params.basemap deprecated since version 4.13. Use view instead.
-*	The light-blue, dark-blue, light-green, dark-green, light-purple, dark-purple, light-red, dark-red are deprecated since 4.19. Please use light or dark instead, or create your own theme.
```

Comment:
these should still be included I think

Reply:
They seem to all be there. I copied over the 4.21 release notes, so everything got included and then alphabetized.

Reply:
Resolved and did some minor reorg.